2019-10-22 12:47:39,996 - INFO - Beginning execution of write_pops_file.py in directory /u/home/j/jonmah/lohmueller_lab/Scripts

2019-10-22 12:47:39,996 - INFO - Progress is being logged to ../Data/AW_to_neutral/seed_5_syn_write_pops_log.log

2019-10-22 12:47:39,997 - INFO - Parsed the following arguments:
	num_ind_2 = 0
	outprefix = ../Data/AW_to_neutral/seed_5_syn
	num_ind_1 = 8

2019-10-22 12:47:40,002 - INFO - Finished writing ../Data/AW_to_neutral/seed_5_syn_pops_file.txt.
2019-10-22 12:47:40,116 - INFO - Beginning execution of write_pops_file.py in directory /u/home/j/jonmah/lohmueller_lab/Scripts

2019-10-22 12:47:40,116 - INFO - Progress is being logged to ../Data/AW_to_neutral/seed_5_nonsyn_write_pops_log.log

2019-10-22 12:47:40,117 - INFO - Parsed the following arguments:
	num_ind_2 = 0
	outprefix = ../Data/AW_to_neutral/seed_5_nonsyn
	num_ind_1 = 8

2019-10-22 12:47:40,120 - INFO - Finished writing ../Data/AW_to_neutral/seed_5_nonsyn_pops_file.txt.
Traceback (most recent call last):
  File "fitdadi_infer_DFE.py", line 480, in <module>
    DemographicAndDFEInference().main()
  File "fitdadi_infer_DFE.py", line 268, in main
    parser = inferDFEParser()
NameError: global name 'inferDFEParser' is not defined
